heartbeat/magefile.go

```diff
@@ -93,3 +94,17 @@ func Update() error {
 func Fields() error {
 	return mage.GenerateFieldsYAML("monitors/active")
 }
+
+// GoTestUnit executes the Go unit tests.
+// Use TEST_COVERAGE=true to enable code coverage profiling.
+// Use RACE_DETECTOR=true to enable the race detector.
+func GoTestUnit(ctx context.Context) error {
```

Comment:
It's a bit a pity that each Beat needs to have all commands inside. Is there a way that mage could "inherit" commands?

Reply:
Hopefully in the future this is supported by mage: https://github.com/magefile/mage/issues/108